src/wazuh_modules/vulnerability_scanner/src/scanOrchestrator/resultIndexer.hpp

```diff
@@ -54,14 +54,7 @@ class TResultIndexer final : public AbstractHandler<std::shared_ptr<TScanContext
             for (const auto& [key, value] : data->m_elements)
             {
                 logDebug2(WM_VULNSCAN_LOGTAG, "Processing and publish key: %s", key.c_str());
-                if (value.contains("operation") && value.contains("id"))
-                {
-                    m_indexerConnector->publish(value.dump());
-                }
-                else
-                {
-                    logError(WM_VULNSCAN_LOGTAG, "Invalid element to publish: %s", value.dump().c_str());
-                }
+                m_indexerConnector->publish(value.dump());
```

Comment:
Why remove the contains for operation and id ?

Reply:
A poor job restoring the changes